### ISSUE TITLE
helm-do-ag-this-file strange behaviour

### ISSUE THREAD

**Issue by gabrielsimoes** [syohex/emacs-helm-ag#209]
When pressing RET on an entry when running helm-do-ag-this-file, it doesn't go to the selected line. Also, it doesn't highlight lines when follow mode is enabled as it does when running helm-do-ag. Thanks in advance!

**Comment by syohex** [comment]
Did you use latest version ? helm-ag had issue about it, but I fixed it at #202.
And I cannot reproduce this issue.

![this-file](https://cloud.githubusercontent.com/assets/554281/12407453/d59f246c-be9a-11e5-9d34-b7cc87bbfffa.gif)

**Comment by gabrielsimoes** [comment]
Playing with my configuration, I discovered using `pt` instead of `ag` is causing this.

``` elisp
(custom-set-variables
       '(helm-ag-base-command "pt -S -e --nocolor --nogroup"))
```

**Comment by syohex** [comment]
Thanks for information. I have fixed at #211. Please check latest version.

**Comment by gabrielsimoes** [comment]
It is working now, thanks!
